### ISSUE TITLE
.NET core example

### ISSUE THREAD

**Issue by JadaVonRuth** [obfuscar/example#11]
Please provide a working .net core 3.1+ example that would work with dotnet build/pack/publish command, stackoverflow says it's possible to use for .net core but its not shown anywhere how. Ideal would be an example for nuget package project.

**Comment by lextm** [comment]
If your project uses `dotnet publish` to generate binaries for deployment, then you can follow

https://github.com/obfuscar/obfuscar/issues/132#issuecomment-371606089

**Comment by JadaVonRuth** [comment]
No, we are using dotnet build + dotnet pack via jenkins build pipeline. I just added it there for completess.

**Comment by lextm** [comment]
Obfuscar does not support automatic .NET Core assembly resolution, so `dotnet pack` won't work well with it. No plan to improve in that area but pull requests are welcome.

**Comment by JandosKh** [comment]
I tried "dotnet publish --self-contained". The problem is it produces platform-specific executables, as explained in the [documentation](https://docs.microsoft.com/en-us/dotnet/core/deploying/#publish-self-contained).

**Comment by lextm** [comment]
@JandosKh ，analyze each of the generated files please, and you will see the platform specific files are irrelevant when your goal is to obfuscate managed assemblies.

**Comment by lextm** [comment]
.NET 10 examples were added in Jan.
